### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by gwwatkin**
Same as [LSC's](https://github.com/latticesurgery-com/lattice-surgery-compiler/tree/dev/.github/ISSUE_TEMPLATE).

### CODE REVIEW

**Review by alexnguyenn — COMMENTED** [review]
Might want to update the bug report template since it is written for Python

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+How to Reproduce
+----------------
+
+
+If possible, please provide us with the steps that lead to the behavior that you are seeing, and include the snippet of the code that generates the error as well as the error message from python below. 
+
+### Code Snippet
+
+```python
```
Comment (gwwatkin) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]:
```suggestion
```ts
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Issue Description
+-----------------
+
+Insert a short description of the bug here, along with what you expected the behavior to be.
+
+How to Reproduce
+----------------
+
+
+If possible, please provide us with the steps that lead to the behavior that you are seeing, and include the snippet of the code that generates the error as well as the error message from python below.
```
Comment (gwwatkin) [inline .github/ISSUE_TEMPLATE/bug_report.md:19]:
```suggestion
If possible, please provide us with the steps that lead to the behavior that you are seeing, and include the snippet of the code that generates the error as well as the error message from the browser's or node's console below. 
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+Environment Context
+-------------------
+
+Please add information about your operating system, python version and dependencies here. 
+
+```
+Copy and paste the output of pip freeze or conda list here.
+```
```
Comment (alexnguyenn) [inline .github/ISSUE_TEMPLATE/bug_report.md:40]:
```suggestion
```
I don't think we would need this part.

Comment (alexnguyenn) [reply .github/ISSUE_TEMPLATE/bug_report.md:40]: Also maybe add a screenshot section under the How to Reproduce section - I think that is more important for web applications.

**Review by alexnguyenn — APPROVED** [review]

**Review by Keelando — COMMENTED** [review]
So there should be something on the home page of the website indicating that we are accepting suggestions and requests, right? If we ever get traffic, we could set up an email list to contributors.

**Pull request merged**